service/api/src/main/java/org/eclipse/kapua/KapuaErrorCodes.java

```diff
@@ -97,4 +97,9 @@
      */
     MAX_NUMBER_OF_ITEMS_REACHED,
 
+    /**
+     * End before start time error
+     */
+    END_BEFORE_START_TIME_ERROR
```

Comment:
This is not the appropriate class where to put this value. Please place it near the `kapua-trigger` module

Reply:
Hi. I tryed to place this near the kapua-trigger module, but when I do this I can't add reference on this error code because code is not reusable.

Reply:
You are right! Then is ok like that.